catalyst-gateway/bin/src/db/index/schema/mod.rs

```diff
@@ -16,8 +16,11 @@ use crate::{settings::cassandra_db, utils::blake2b_hash::generate_uuid_string_fr
 /// This constant is ONLY used by Unit tests to identify when the schema version will
 /// change accidentally, and is NOT to be used directly to set the schema version of the
 /// table namespaces.
-#[allow(dead_code)]
-const SCHEMA_VERSION: &str = "08193dfe-698a-8177-bdf8-20c5691a06e7";
+pub const SCHEMA_VERSION: &str = "08193dfe-698a-8177-bdf8-20c5691a06e7";
+
+/// AWS Keyspace actions are not instant, induce sleep to wait for remote keyspace
+/// actions to take place.
+const KEYSPACE_LATENCY: u64 = 15;
```

Comment:
Should be an env var, and should default to 0.
Don't call it KEYSPACE_LATENCY, call it something like CASSANDRA_{name}_KEYSPACE_CREATION_DELAY_SECS.
Don't default it to 15, if thats needed for production, it should be configured as required in production.
In development its not required, hence why it should default to 0.

Volatile and Immutable DB's may need different options, so this option should be divided.

What would be better here would be that we don't have any delay, and we poll the DB to see when the keyspace has been created.  An arbitrary delay can easily break in production or needs to be set arbitrarily large.
See: https://docs.aws.amazon.com/keyspaces/latest/devguide/keyspaces-create.html
For instructions on how to do that.